### PR TITLE
🌱 Make VirtualMachineClass configurable when using vcsim

### PR DESCRIPTION
**Pull request by fabriziopandini**
**What this PR does / why we need it**:
Make VirtualMachineClass configurable when using vcsim

### CODE REVIEW

File: test/infrastructure/vcsim/controllers/vcsim_controller.go [hunk 1]
```diff
@@ -286,7 +240,62 @@ func (r *VCenterSimulatorReconciler) reconcileNormal(ctx context.Context, vCente
...
+			StoragePolicy: vcsimDefaultStoragePolicyName,
+
+			// Those are settings for a fake content library we are going to create given that it doesn't exists in vcsim by default.
+			// It contains a single dummy image.
+			ContentLibrary: vmoperator.ContentLibraryConfig{
+				Name:      "vcsim",
+				Datastore: vcsimDatastorePath(datacenter, datastore),
+				Item: vmoperator.ContentLibraryItemConfig{
+					Name: "vcsim-default-image",
+					Files: []vmoperator.ContentLibraryItemFilesConfig{ // TODO: check if we really need both
```
Comment (chrischdi) [inline test/infrastructure/vcsim/controllers/vcsim_controller.go:274]: TODO still there by accident or on purpose? :-)

**Review by chrischdi — COMMENTED** [review]
One open question

as of that

/lgtm

**Comment by sbueringer** [comment]
/lgtm

pending the open question

File: test/infrastructure/vcsim/controllers/vcsim_controller.go (hunk 1 above)
Comment (fabriziopandini) [reply test/infrastructure/vcsim/controllers/vcsim_controller.go]: it was a left-over, cleaned up

**Pull request merged**